### PR TITLE
Join a local

### PR DESCRIPTION
**Pull request by shushugah**
This creates `/chapters` as seen [here](https://deploy-preview-256--techworkersco.netlify.app/chapters/). 

Improves findability for humans and search engines, for individual chapters, and allows extra information to be included, for example Facebook/Twitter/Meetup etc... 

This also reduces a lot of duplicate text/clutter, and provides a cleaner header. 

Some ToDos remain:
- [ ] Translate each city name, and "find links at..."
- [ ] Improve styling within Chapters

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -33,43 +33,6 @@ exclude:
   - .jekyll-cache/
 languages: ["en", "es", "fr", "it", "pt", "ru"]
 exclude_from_localizations: ["assets"]
-chapters:
```
Comment (rossPatton) [inline _config.yml:36]: why delete this? iirc this is what results in an empty chapters dropdown menu

if we want to remove that for some reason, i think we should just remove it entirely and add the chapters link to the hamburger menu

**Review by rossPatton — CHANGES_REQUESTED** [review]
just have some questions about the changes to the navigation

File: _config.yml (hunk 1 above)
Comment (shushugah) [reply _config.yml:36]: Imo, also for tracking purposes, would be better if people opened/went to the chapters page itself. I think adding this to top of "site.links" would be good (the main hamburger menu), and remove/hide the social media links in the top hamburger menu, while leaving the footer the same

File: _includes/header.html [hunk 2]
```diff
@@ -61,30 +61,8 @@ <h1 class="title">
           role="navigation"
           class="navOverlay abs hide list-style-none">
           <li class="p-2 border-b-1">
-            Join a Chapter
+            <a href="/chapters">Join a Chapter</a>
```
Comment (shushugah) [inline _includes/header.html:64]: @rossPatton this here, changes the navigation to be flat/simple link. Could also merge to be at top of the main hamburger menu

Comment (rossPatton) [reply _includes/header.html:64]: Right, what i'm suggesting is we just move that link into the hamburger menu, and just remove this link/subnav

**Review by rossPatton — APPROVED** [review]

**Pull request merged**